src/App.jsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import Footer from "./components/Footer";
 
 function App() {
-  return <div>Basic structure</div>;
+  return (
+    <div>
+      <Footer />
```

Comment:
i'm not sure if Footer should be called in App.jsx or in Home.jsx. For Navbar component we called it in the home page and for every other page, but i'm not sure which one is the correct way